ts/features/payments/bizEventsTransaction/components/PaymentsBizEventsTransactionHeadingSection.tsx

```diff
@@ -87,11 +89,15 @@ export const PaymentsBizEventsTransactionHeadingSection = ({
         onPress={handlePressTransactionDetails}
       />
       <VSpacer size={8} />
-      <PaymentsBizEventsTransactionTotalAmount
-        loading={isLoading}
-        totalAmount={calculateTotalAmount(transactionInfo)}
-      />
-      <VSpacer size={8} />
+      {totalAmount && (
+        <>
+          <PaymentsBizEventsTransactionTotalAmount
+            loading={isLoading}
+            totalAmount={totalAmount}
+          />
+          <VSpacer size={8} />
+        </>
+      )}
       <FeeAmountSection />
```

Comment:
I see that calculateTotalAmount has this logic:

```ts
if (!transactionInfo || !transactionInfo.amount || !transactionInfo.fee) {
    return undefined;
 }
```
So you'll have an `undefined` value also if you have the amount but not the fees. Am I right? 
In this case you'll hide the total amount part **but** not the fee section. What do you think?

Reply:
Good catch! Thank you for notifying this logical issue! What do you think about the changes addressed in the following commit: df9882ea121df14426bebab1a2aa085b9c06ad41